app.py

```diff
@@ -8,13 +11,23 @@ def check_query(resp):
             print(f'+- {m.uri}: {m.scores["cosine"].value:.6f}, {m.tags}')
 
 def main():
-    docs = DocumentArray(from_files('toy-data/*.mp3'))
-
+    cur_dir = os.path.dirname(os.path.abspath(__file__))
+    workspace = os.path.join(cur_dir, 'workspace')
+    logger = logging.getLogger('audio-search')
+    docs = DocumentArray.from_files('toy-data/*.mp3')
+    if os.path.exists(workspace):
+            logger.error(
```

Comment:
let's just print because we are not logging to any file
```suggestion
            print(
```